### ISSUE TITLE
Bug in syntax-rules expander: lookup of ‘...’ flawed

### ISSUE THREAD

**Issue by mnieper** [larcenists/larceny#780]
The following program should print `...`. However, Larceny raises an error `Too many ...'s`.

``` scheme
(import (rename (scheme base))
        (scheme write))

(let ((... 'foo))
  (define-syntax bar
    (syntax-rules ()
      ((bar)
       (begin
     (display '...)
     (newline)))))

  (bar))
```

The normal binding of `...` is exported by `(scheme base)`, which, however, should be shadowed inside the `let` binding construct.

**Comment by WillClinger** [comment]
See also ticket #747.

**Comment by WillClinger** [comment]
Fixed by changeset fb44ba73d25f7380b3766109314c5f16bb55e34e
